### PR TITLE
Bradley/app top bar

### PR DESCRIPTION
**Pull request by BradleyLi**
[The](url) UI part of the app top bar is completed, with only a few issues that I am struggling to figure out:
1. I am having trouble hiding the error and trash icon when the display is in the mobile version;
2. I don't know if I should also implement the logic for the more_vert and error, trash icon, as when clicking the more _vert icon, it should show the error and trash options.

### CODE REVIEW

**Comment by BradleyLi** [comment]
Mobile view:
Screenshot 2023-02-15 at 11 24 59 PM

Web view:
Screenshot 2023-02-15 at 11 27 02 PM

**Comment by tim-macphail** [comment]
Nice job Bradley! I'll leave a few tips + comments

File: client/src/components/AppTopBar/AppTopBar.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <div className="flex-1">
+          <h1 className="text-4xl mt-auto text-on-surface font-headline font-bold md:text-2xl">
+            {format(courseId)}
+          </h1>
+          <h2 className="text-base font-normal text-outline">
+            {description}
+          </h2>
+        </div>
+        
+        <span className={classnames("material-symbols-outlined", "md:inline", "hidden")}>error</span>
```
Comment (tim-macphail) [inline client/src/components/AppTopBar/AppTopBar.tsx:25]: Wrap these spans in Buttons to make them clickable

Comment (BradleyLi) [reply client/src/components/AppTopBar/AppTopBar.tsx:25]: Thanks for the feedback, Tim! Fixed it now! By the way, should I also implement the onClick callback function for these two span icons?

File: client/src/components/AppTopBar/AppTopBar.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+interface AppTopBarProps extends HTMLAttributes<HTMLDivElement> {
+  courseId: string | undefined;
+  description: string | undefined;
+}
+
+const AppTopBar = ({ courseId, description }: AppTopBarProps) => {
+  const format = (courseId: string | undefined) => courseId?.replace("-", " ").toUpperCase();
+
+  // return (
```
Comment (cloudyyoung) [inline client/src/components/AppTopBar/AppTopBar.tsx:13]: Please remove commented code snippets.

**Review by cloudyyoung — APPROVED** [review]
LGTM!

**Pull request merged**